src/components/ParticipantList.jsx

```diff
@@ -54,11 +59,51 @@ export default function ParticipantList({ studyID, title, onParticipantSelect, s
         })()
     }, [participants])
 
+
     let addParticipant = async () => {
         let newCount = state.newCount
         let ids = []
-        for (let i = 0; i < newCount; i ++)
-            ids = [...ids, (await LAMP.Participant.create(studyID, { study_code: '001' })).data]
+        let id = []
+        let tempPassword = []
+
+        for (let i = 0; i < newCount; i ++) {
+            id = (await LAMP.Participant.create(studyID, { study_code: '001' })).data
+            tempPassword = id
+            if (!!(await LAMP.Credential.create(id, tempPassword, 'Temporary Login')).error) {
```

Comment:
You lost a parameter here: 
```
await LAMP.Credential.create(id, data.emailAddress, data.password, data.name)
```

You have: 
```
await LAMP.Credential.create(id, tempPassword, 'Temporary Login')
```

So your password will become the email address and `Temporary Login` will become the password and you're not naming the credential anymore. Instead, just after the `id, ` make sure you add: 
```
`${id}@lamp.com`,
```